wolfssl/openssl/asn1.h

```diff
@@ -164,7 +164,7 @@ WOLFSSL_API int wolfSSL_ASN1_item_i2d(const void *src, byte **dest,
                                       const WOLFSSL_ASN1_ITEM *tpl);
 
 /* Need function declaration otherwise compiler complains */
-// NOLINTBEGIN(readability-named-parameter)
+/* NOLINTBEGIN(readability-named-parameter) */
```

Comment:
I think lint requires the ‘/* // NOLINT */‘. Check with @douzzer  for clarification

Reply:
Confirmed with douzzer that /* // NOLINT */ is required, updated.